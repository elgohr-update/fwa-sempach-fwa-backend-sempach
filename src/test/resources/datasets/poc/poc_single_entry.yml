poc:
 - id: 1
   name: test
   date: 2018-03-05