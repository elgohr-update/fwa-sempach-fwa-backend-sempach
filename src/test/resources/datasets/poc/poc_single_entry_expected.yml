poc:
 - id: 2
   name: test2
   date: 2018-03-13